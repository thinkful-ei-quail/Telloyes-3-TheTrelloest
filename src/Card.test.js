import Card from "./Card";
import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import List from "./List";

describe("Card", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it("renders to UI as expected", () => {
    const tree = renderer
      .create(<Card title="Something" content="something else" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
