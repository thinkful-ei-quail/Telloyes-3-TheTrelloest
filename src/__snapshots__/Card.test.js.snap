// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders to UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    something
  </h3>
  <p>
    someotherthing
  </p>
</div>
`;
